hotbar.coffee

```diff
@@ -92,16 +94,28 @@ class InventoryHotbarClient extends InventoryHotbarCommon
   enable: () ->
     @inventoryWindow.container.style.visibility = ''
     @onSlots = {}
+    @console.on 'open', =>
+      @scrollLock = false;
+    @console.on 'close', =>
+      @scrollLock = true;
 
     if @wheelEnable
       ever(document.body).on 'mousewheel', @mousewheel = (ev) => # TODO: also DOMScrollWheel for Firefox
-        console.log 'mousewheel',ev
+        if not @scrollLock then return else
+        # console.log 'mousewheel',ev
         delta = ev.wheelDelta
         delta /= @wheelScale
-        delta = Math.floor delta
+        delta = Math.floor delta // 120
+        if delta > 0
+          delta = 1
+        else if delta < 0
+          delta = -1
         newSlot = @selectedIndex + delta
-        newSlot = newSlot %% @inventoryWindow.width
-        console.log newSlot
+        if(newSlot == @inventoryWindow.width)
```

Comment:
Oh wait i see, I didnt understand what that was doing before. I'll do some testing